src/main/java/converter/Conversion/StarterMetaDataConversion.java

```diff
@@ -135,18 +129,11 @@ public static void main(String[] args) throws Exception {
 
                 if (!flagError) {
                     try {
-                        if (!metsOutput.write(procFile.getAbsolutePath())) {
-
-                            logger.error("File " + procFile.getAbsolutePath() + " couldn't be written in mets format");
-
-                            rollbackLog.info(newFile.getAbsolutePath() + " - Mets couldn't be saved - processing cancelled");
-                            flagError = true;
-                        } else {
+                        ((FileformatInterface) metsOutput).write(procFile.getAbsolutePath());
```

Comment:
Same here: removing if condition without adjusting the indent of previous condition block.

Reply:
Same as above.